src/BudgetManager.Shared/BudgetManager.Shared.DataAccess.MongoDB/DbContexts/MongoDbContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+
+namespace BudgetManager.Shared.DataAccess.MongoDB.DbContexts
+{
+    public class MongoDbContext : IMongoDbContext
+    {
+        private readonly MongoClient mongoClient;
+        private readonly IMongoDatabase mongoDatabase;
```

Comment:
StyleCop seems to be complaining in few places

Reply:
Sorry, I wasn't using StyleCop. How to enable StyleCop to detect issues like this?